appscale/tools/appscale_tools.py

```diff
@@ -215,29 +227,193 @@ def add_keypair(cls, options):
 
   @classmethod
   def describe_instances(cls, options):
```

Comment:
this may be misleading as a name, since we have a bunch of describe_instances in the agents,,,